app/views/partials/inputDropdown.scala.html

```diff
@@ -25,7 +25,9 @@ <h2 class="govuk-label-wrapper">
             </label>
         </h2>
         <div id="@formFieldId-hint" class="govuk-hint">
-            @fieldDescription
+             @* hard coding the description as the database cannot contain html*@
```

Comment:
Will this not override the any dropdown field that does not have a description? Think it would need to include an if statement to just apply the additional content for the FOI exemption code field?

Reply:
You're right. I've added an if statement now.